### PR TITLE
Fixing the code style and beginning week-3 starter project work

### PR DESCRIPTION
**Pull request by clayferg**

### CODE REVIEW

File: portfolio/.github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      ################################
+      # Run Linter against code base #
+      ################################
+      - name: Lint Code Base
+        uses: docker://github/super-linter:v2.2.0
+        env:
+          VALIDATE_ALL_CODEBASE: false
```
Comment (ciddu) [inline portfolio/.github/workflows/linter.yml:47]: should VALIDATE_ALL_CODEBASE be true?

**Review by ciddu — CHANGES_REQUESTED** [review]

File: portfolio/.github/workflows/linter.yml (hunk 1 above)
Comment (clayferg) [reply portfolio/.github/workflows/linter.yml]: Sure, I'll try that, I wasn't completely sure what to do with the file.

**Review by clayferg — COMMENTED** [review]
The two previous commits only became visible (to me at least) after I did a git pull for the branch. I think I did something wrong with how I added the linter to the repo because the difficulties seemed to start after that. Do you know by any chance what is likely causing this?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-react-app": "^5.2.1",
+    "eslint-plugin-flowtype": "^4.7.0",
+    "eslint-plugin-import": "^2.21.2",
+    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-react": "^7.20.0",
+    "eslint-plugin-react-hooks": "^2.5.1"
+  }
+}
```
Comment (ciddu) [inline package.json:41]: No new line at end of the file.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    "url": "git+https://github.com/clayferg/clayferg-projects.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/clayferg/clayferg-projects/issues"
+  },
+  "homepage": "https://github.com/clayferg/clayferg-projects#readme",
+  "dependencies": {
```
Comment (ciddu) [inline package.json:24]: Add "eslint": "^6.8.0" to dependencies

https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee

**Review by ciddu — CHANGES_REQUESTED** [review]
One suggestion in general regarding code reviews. I see three different types of changes in the same pull request.

1) Lint capability for codebase.
2) Code styling.
3) Week 3 starter project.

While optional for this PR, for any future changes lets split changes not related to each other into multiple PRs. That way it's easier to review and also track changes.

File: package.json (hunk 2 above)
Comment (clayferg) [reply package.json:42]: Alright, thanks.

File: package.json (hunk 3 above)
Comment (clayferg) [reply package.json:24]: Cool, added in the commit below.

**Comment by ciddu** [comment]
ESLint still fails.

**Review by ciddu — APPROVED** [review]
Fix the lint errors before merging.  If the source-root fix didn't help, let me know.

**Pull request merged**